ts/store/reducers/onboarding.ts

```diff
@@ -12,17 +13,22 @@ import { GlobalState } from "./types";
 export type OnboardingState = Readonly<{
   isTosAccepted: boolean;
```

Comment:
this is not needed anymore, since it's equal to `acceptedTosVersion === config.tosVersion`